app/src/androidTest/java/org/mozilla/tv/firefox/helpers/CustomPocketFeedStateProvider.kt

```diff
@@ -20,22 +21,22 @@ import org.mozilla.tv.firefox.utils.BuildConfigDerivables
  */
 class CustomPocketFeedStateProvider {
 
-    private val pocketEndpoint = object : PocketEndpoint("VERSION", "www.mock.com".toUri()) {
+    private val localeIsEnglish: () -> Boolean = { true }
```

Comment:
We have a unit test for non-EN locales, and it's simple enough logic that I don't think we need an Espresso test for it.  This is just a dependency for a few of these objects, so I wanted to declare it in one place instead of two.